src/client/power-pages/create/utils/CommonUtils.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is it specific to only create. I would say create a commonUtil

Reply:
Will check for possibility: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3226760